script/deploy/gnosisSAFE/proposeTx.ts

```diff
@@ -50,15 +50,17 @@ const main = async () => {
   console.info('SAFE Address: ', safeAddress)
   console.info('Diamond Address: ', diamondAddress)
 
+  // Parse the raw diamond cuts
+  const cuts = JSON.parse(rawCuts)
+
+  console.info('Cuts: ', cuts)
```

Comment:
Wanna keep this in there? I dont really mind, up to you

Reply:
Will keep for now just for debugging.